Documentation/HowToGuides/AddingACollisionIgnorer/README.md

```diff
@@ -0,0 +1,64 @@
+# Adding A Collision Ignorer
+
+> * Level: Beginner
+>
+> * Reading Time: 5 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+The `Input Mutator ColliisonIgnorer` provides a way of allowing two or more objects to not interact with each other.
+
+## Let's Start
+
+### Step 1
+
+Create a new `Cube` Unity 3D Object by selecting `Main Menu -> GameObject -> 3D Object -> Cube` and change the `Transform` properties to:
+
+* Position: `X = 0`, `Y = 0`, `Z = 0`
+
+![Adding A Cube](assets/images/AddingACube.png)
+
+### Step 2
+
+Create another new `Cube` Unity 3D Object by selecting `Main Menu -> GameObject -> 3D Object -> Cube` and change the `Transform` properties to:
+
+* Position: `X = 0`, `Y = 2`, `Z = 0`
+
+![Adding The Second Cube](assets/images/AddingTheSecondCube.png)
+
+### Step 3
+
+Click the `Add Component` button and select the `Rigidbody` component to the second `Cube`. 
+
+![Adding A Rigidbody](assets/images/AddingARigidbody.png)
+
+Play the `Unity Scene` and you should see the second `Cube` land on top of the first `Cube`.
```

Comment:
Use the cube name to identify:

Play the `Unity Scene` and you should see `Cube (1)` land on top of `Cube` because the colliders on each cube will prevent them passing through each other.